### DISCUSSION TITLE
Welcome to BewlyBewly Discussions!

### DISCUSSION THREAD

**Discussion by hakadao** [BewlyBewly/BewlyBewly#108] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bigmanBass666** [comment]
爱死了, B站页面好看炸了❤❤❤❤❤❤❤❤❤❤

**Comment by lgx777** [comment]
作者大佬，我目前使用下来的一个小问题就是顶栏里的收藏夹一栏里文字相比其他栏有点糊，不太清晰，来反馈一下。
